scripts/build-windows-installer.ps1

```
@@ -190,9 +190,17 @@ function Get-TrustedTimestampUrl {
     return $uri.AbsoluteUri
 }
 
-function Sign-SetupIfConfigured([string]$SetupPath, [string]$BuildRoot) {
+function Sign-FilesIfConfigured([string[]]$Paths, [string]$BuildRoot) {
```

PSScriptAnalyzer (1.25.0)

[warning] The cmdlet 'Sign-FilesIfConfigured' uses an unapproved verb.

(PSUseApprovedVerbs)